modules/talos/manifests/init.pp

```diff
@@ -23,7 +23,7 @@
                     ensure  => directory,
                     owner   => $user,
                     group   => 'staff',
-                    mode    => '0755',
+                    mode    => '0777',
```

Comment:
do these need to be wide open, or can they be made only writable by the generic-worker users' group or main user?